packages/web-app-files/src/components/SideBar/Shares/Collaborators/ShowCollaborator.vue

```diff
@@ -265,10 +265,16 @@ export default {
     },
 
     $_reshareInformation() {
-      if (!this.collaborator.resharers) {
+      try {
+        this.collaborator.resharers.forEach(function(share) {
+          if (typeof share.displayName !== 'string' || !share.displayName) {
+            throw Error('displayName of resharer is not a string')
```

Comment:
@kulmann @fschade let's discuss if we should fall back on this behaviour more often and if so document it as best practice/convention :)